tools/pythonpkg/tests/fast/test_2747.py

```diff
@@ -0,0 +1,82 @@
+import duckdb
+import pandas as pd
+import numpy as np
+import datetime
+from decimal import Decimal
+from uuid import UUID
+
+def get_all_types():
+	conn = duckdb.connect()		
+	all_types = conn.execute("describe select * from test_all_types()").fetchall()
+	types = []
+	for cur_type in all_types:
+		types.append(cur_type[0])
+	return types
+
+all_types = get_all_types()
+
+class Test2747(object):
```

Comment:
Now that enums are stored in vectors, this creates new and exciting ways in which things can break. Can we add a SQL test that has large enum values (i.e. non-inlined strings as enum values)?